### PR TITLE
Updated 3D tracks finder & weighted CNR

### PR DESCRIPTION
**Pull request by lzambell**

### CODE REVIEW

**Review by YoannKermaidic — COMMENTED** [review]
Just tested the PR - without issues.

Question before approval on my side: 

Is the weighted CNR procedure well setup in the current PR (having an option to enable/disable it depending on the analysis type - TDE/BDE/else, weight = capa or rawRMS)? 

Or shall we discuss the way to select a specific method via the analysis settings before merging? This to make sure we know what we apply.

**Comment by lzambell** [comment]
I've changed that in the latest commit : TDE needs 'regular' CNR, and BDE needs the 'per_view' one. 
At the moment, the capa weight is done only in the 'per_view' case, although with the new BDE data, we might not need anymore.

**Comment by lzambell** [comment]
it's now set in the analysis parameters

File: det_spec.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def configure(detector, period, elec, run):
         cf.n_tot_channels = int(data[elec]['n_tot_channels'])
         cf.n_sample = int(data[elec]['n_sample'])
         cf.sampling = float(data[elec]['sampling'])
-        cf.ADC_to_fC = float(data[elec]['ADC_to_fC'])
+        cf.ADC_per_fC = float(data[elec]['ADC_per_fC'])
```
Comment (olantwin) [inline det_spec.py:20]: Would be better to squash this change with the previous one to not break things.

File: settings/geo_coldbox.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
 	    "n_tot_channels":"1792",
 	    "sampling":"2",
 	    "n_sample":"8192",
-	    "ADC_to_fC":"1",
+	    "ADC_per_fC":"266.76",
```
Comment (olantwin) [inline settings/geo_coldbox.json]: This change also should probably be merged with the commit where you rename the variable.

**Review by olantwin — APPROVED** [review]
Looks good to me.
Maybe a good opportunity to learn some of the advanced rebasing skills? ;)

**Pull request merged**